packages/design-system/src/components/icon-button.tsx

```diff
@@ -66,8 +55,44 @@ export const IconButton = styled("button", {
         width: "$8",
       },
     },
+    variant: {
```

Comment:
this is a big change to the design system, we need to think through what are all the variants we have and make it all consistent ... lets have a discussion on this ... seems like a major pain point with all the design system stuff right now

Reply:
Removed